src/pages/de/guides/integrations-guide.md

```diff
@@ -0,0 +1,132 @@
+- Tools wie Tailwind und Partytown mit wenigen Zeilen Code integrieren.
+- Neue Features zu deinem Projekt hinzufügen, wie z.B. automatische Sitemap-Generierung.
+- Custom Code schreiben, der sich in den Erzeugungsprozess, den Entwicklungs-Server und mehr einhängen lässt.
+
+## Offizielle Integrationen
+
+<IntegrationsNav />
+
+## Autmatische Einrichtung von Integrationen
+
+Astro stellt ein `astro add` Kommando zur Verfügung, um das Einrichten von Integrationen zu automatisieren.
+
+:::Achtung
+Wir werden immer um Bestätigung bitten, bevor wir deine Dateien aktualisieren. Jedoch ist es empfehlenswert, ein Versionskontroll-Backup zu haben.
+:::
+Führe das `astro add` Kommando mit dem Paketmanager deiner Wahl aus und unser automatischer Integration-Wizard wird deine Konfigurationsdatei aktualisieren und alle notwendigen Abhängigkeiten installieren.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npx astro add react
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpx astro add react
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro add react
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Es ist sogar möglich mehrere Integrationen gleichzeitig zu konfigurieren!
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npx astro add react tailwind partytown
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpx astro add react tailwind partytown
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro add react tailwind partytown
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+:::note[Abhängigkeiten von Integrationen managen]
+Solltest du eine Warnung wie `Cannot find package '[package-name]'` erhalten, dann hat dein Paketmanager vermutlich die [peer dependencies](https://nodejs.org/en/blog/npm/peer-dependencies/) installiert. Um diese zu installieren, führe einfach `npm install [package-name]` aus.
+:::
+
+## Integrationen nutzen
+
+Astro-Integrationen werden immer über die `integrations` Option in deiner `astro.config.mjs` Datei konfiguriert.
+
+Es gibt drei Arten, um Integrationen zu importieren:
+1. Eine Integration über ein npm-Paket installieren.
+2. Deine eigene Integration über eine lokale Datei innerhalb deines Projekts importieren. 
+2. Eine eigene Integration direkt in der Konfigurationsdatei schreiben.
+
+```js
+// astro.config.mjs
+import {defineConfig} from 'astro/config';
+import installedIntegration from '@astrojs/vue';
+import localIntegration from './my-integration.js';
+
+export default defineConfig({
+  integrations: [
+    // 1. Aus einem installiertem npm-Paket importieren
+    installedIntegration(),
+    // 2. Aus einer lokalen JS-Datei importieren
+    localIntegration(),
+    // 3. Ein Objekt
+    {name: 'namespace:id', hooks: { /* ... */ }},
+  ]
+})
+```
+
+Schaue dir die [Integration-Schnittstelle](/en/reference/integrations-reference/) an, um mehr darüber zu erfahren, wie du deine eigenen Integrationen erstellen kannst.
```

Comment:
```suggestion
Sieh dir die [Integrations-API](/de/reference/integrations-reference/) an, um mehr darüber zu erfahren, wie du deine eigenen Integrationen erstellen kannst.
```

We're already using the term API in various places, so it should be safe to not translate it here.